lib/README.md

```diff
@@ -56,6 +58,16 @@ It returns a [HalResource Object](#halresource-object), encapsulating the origin
 | `isPropertyRequired`     | `(String)=>Bool`           | Receives the key of a property. Returns true if it exists within the "\_options.required" array.                                    |
 | `getSchemaProperties`    | `()=>Object[]`             | Returns an array of the original schema objects for properties (adding "key"). Taken from the "\_options.properties" object.        |
 | `getSchemaProperty`      | `(String)=>Object`         | Receives the key of a property. Returns the property's schema object (adding "key") if exists within "\_options.properties" object. |
+| `addLink`      | `(Object)=>void`         | Receives an object as parameter. This object can contain the usual link properties as { rel, name, href, title } where rel and href are required. The obect will be added to the  "\links" section using rel as key  |
```

Comment:
Remove the name of the attributes other than href and rel. Specify that these 2 are required